### ISSUE TITLE
[QUESTION] Download Config

### ISSUE THREAD

**Issue by EVOTk** [Lissy93/dashy#36]
Hello,
I understand correctly that the modifications made from the browser are not saved in the conf.yml file. and so I thought that this otpions was used to download the current configuration in `Config > Backup Config > Actions > Downloads Config` to save it in the event that we would have made changes from the browser. (Config> Edit Sections for example)

But this is not the case, in any case, this action downloads the original conf.yml file, present in the container.

Is this the normal behavior?

### Exemple :
My conf.yml in contener : ( originaly )
```
appConfig:
  theme: "material-dark"
  fontAwesomeKey: "0821c65656"
pageInfo:
  title: "EVOServ - Dashy"
  navLinks:
    - title: "Accueil"
      path: "/"
    - title: "Code Source"
      path: "https://github.com/Lissy93/dashy"
  description: ""
  footerText: ""
sections:
  - name: "Serveur"
    items:
      - title: "Proxmox"
        description: ""
        url: "https://192.168.2.40:8006"
      - title: "Synology"
        description: ""
        url: "http://192.168.2.42:999"
      - title: "Portainer"
        description: ""
        url: "http://192.168.2.42:9000"
```

In WebBrowser : Config > Edit Sections, add statusCheck for Proxmox, Synology and Portainer :  `statusCheck: true`
In WebBrowser : Backup Config, I can see this:
```
appConfig:
  theme: "material-dark"
  fontAwesomeKey: "0821c65656"
pageInfo:
  title: "EVOServ - Dashy"
  navLinks:
    - title: "Accueil"
      path: "/"
    - title: "Code Source"
      path: "https://github.com/Lissy93/dashy"
  description: ""
  footerText: ""
sections:
  - name: "Serveur"
    items:
      - title: "Proxmox"
        description: ""
        url: "https://192.168.2.40:8006"
        statusCheck: true
      - title: "Synology"
        description: ""
        url: "http://192.168.2.42:999"
        statusCheck: true
      - title: "Portainer"
        description: ""
        url: "http://192.168.2.42:9000"
        statusCheck: true
```

By cons, if clicked `Donwload Config` I download the file conf.yml without the addition of statusCheck, i'm download the originaly conf.yml

**Comment by Lissy93** [comment]
Yeah, you'r right- it's a known issue, and I'm going to re-write the whole config editing process. 
It was initially done in this clumsy way, because it's not possible for a froendend web app to write changes to the disk. But it's now using a Node.js server to serve up the app, so I can now write a node endpoint which can save the conf.yml file to the filesystem. Once that's done the whole process will be much more user friendly

**Comment by Lissy93** [comment]
Oh I think I misunderstood, the Download config button is downloading the wrong config file. Just tested it out, that's a bug - thank you for raising it. I'll get it fixed soon, and let you know when the changes are pushed. In the meantime, you can copy paste the YAML from the text field manually.

**Comment by EVOTk** [comment]
> 
> 
> Oh I think I misunderstood, the Download config button is downloading the wrong config file. Just tested it out, that's a bug - thank you for raising it. I'll get it fixed soon, and let you know when the changes are pushed. In the meantime, you can copy paste the YAML from the text field manually.

Indeed, I am talking about this. Sorry if my explanations are not always clear, I don't speak English.

Thank you for your reply ;)

**Comment by Lissy93** [comment]
Thank you for your patience with this. The issue is now fixed in 7ecb815ec36a6bd33251fb01aa67c03f1de03930 of PR #43. 
When you press the download button, a new file is generated from the current config, and the download should begin automatically :)

**Comment by EVOTk** [comment]
It works perfectly, thank you :)
